docker/release/dockerfiles/opensearch-dashboards.al2.dockerfile

```diff
@@ -3,6 +3,7 @@
 
 
 # This dockerfile generates an AmazonLinux-based image containing an OpenSearch-Dashboards installation.
+# Dockerfile for building an OpenSearch image with name and latest tag
```

Comment:
This should be OpenSearch-Dashboards I think.

Reply:
This can be simplified as '# Dockerfile for building an OpenSearch-Dashboards image'